terraform/main.tf

```diff
@@ -85,6 +85,7 @@ data "template_file" "container_definition" {
     APP_URL                    = var.APP_URL
     USERS_API_URL              = var.USERS_API_URL
     CONTROL_TOWER_URL          = var.CONTROL_TOWER_URL
+    AREAS_API_URL              = https://${data.terraform_remote_state.fw_core.outputs.public_url}/v3/forest-watcher
```

Comment:
`AREAS_API_URL = "https://${data.terraform_remote_state.fw_core.outputs.public_url}/v3/forest-watcher"`

Needs to be in quotes - sorry